themes/V3/5ePHB/style.less

```diff
@@ -795,6 +795,16 @@
 // *****************************
 // *      TABLE OF CONTENTS
 // *****************************/
+
+// Default Exclusions
+.page:has(.frontCover), 
+.page:has(.backCover), 
+.page:has(.insideCover), 
+.page:has(.partCover), 
+.monster,
```

Comment:
Similar issue here. The `--ToC` tag is applied to `.monster`, but the generator only checks the header elements, so this still appears in the TOC.

Reply:
The `--TOC` property *is* cascading to child elements of `< div class="monster"` ... however the test on line 38 is checking for the wrong value (it is testing against `"exclude"`, it should test against `exclude`).